fiftyone/core/runs.py

```
@@ -135,9 +135,10 @@ def from_dict(cls, d):
 
         try:
             config_cls = etau.get_class(config_cls)
-        except:
-            logger.debug(
-                "Unable to load '%s'; falling back to base class", config_cls
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)